Taylor and Francis+NEJM.js

```diff
@@ -164,7 +165,23 @@ function finalizeItem(item, doc, doi, baseUrl) {
 		if (sectionheading.match(/^(Book )?Reviews?$/i))
 			item.tags.push("Book Reviews");
 	}
-
+	
+	// numbering issues with slash, e.g. in case of  double issue "1-2" > "1/2"
+	if (item.issue) item.issue = item.issue.replace('-', '/');
+	
+	//scraping orcid number 
+	let authorSectionEntries = ZU.xpath(doc, '//*[contains(@class, "contribDegrees corresponding ")]');
+	for (let authorSectionEntry of authorSectionEntries) {
+		let authorInfo = authorSectionEntry.textContent; //Z.debug(authorInfo)
+		if (authorInfo.match(/\d+-\d+-\d+-\d+x?/i)) {
+			let authorAffilation = authorInfo.split('Correspondence')[0];
```

Comment:
Seems a far-reaching assumption that "Correpondence" is used as a separator for every T&F journal now and in the future. Maybe rather use `class="entryAuthor"`and the data field of  `class="orcid-author"`?
i.e.
```javascript
let authorSectionEntries = ZU.xpath(doc, '//*[contains(@class, "contribDegrees corresponding ")]');
for (let authorSectionEntry of authorSectionEntries) {
    let authorInfo = authorSectionEntry.querySelector('.entryAuthor');
    let orcidHref = authorSectionEntry.querySelector('.orcid-author');
    if (authorInfo && orcidHref) {
	let author = authorInfo.childNodes[0].textContent;
        let orcid = orcidHref.textContent.replace(/.*(\d+-\d+-\d+-\d+x?)$/i, '$1');
        ...
    }
}
```

Reply:
thanks for your suggestions. I changed the code.